easybuild/easyconfigs/m/matplotlib/matplotlib-3.0.3-intelcuda-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,61 @@
+easyblock = 'PythonBundle'
+
+name = 'matplotlib'
+version = '3.0.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'intelcuda', 'version': '2019a'}
+
+use_pip = True
+check_ldshared = True
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('SciPy-bundle', '2019.03'),
+    ('libpng', '1.6.36'),
+    ('freetype', '2.9.1'),
+    ('Tkinter', '%(pyver)s'),
+]
+
+exts_list = [
+    ('Cycler', '0.10.0', {
+        'modulename': 'cycler',
+        'source_tmpl': 'cycler-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/C/Cycler'],
+        'checksums': ['cd7b2d1018258d7247a71425e9f26463dfb444d411c39569972f4ce586b0c9d8'],
+    }),
+    ('kiwisolver', '1.0.1', {
```

Comment:
@Micket `kiwisolver` 1.1.0 is available now (since today), see https://pypi.org/project/kiwisolver/1.1.0/, so let's update?